src/main.c

```diff
@@ -202,232 +208,234 @@ typedef struct{
   float waveform_r[SE_STATS_GRAPH_DATA];
   float waveform_fps_emulation[SE_STATS_GRAPH_DATA];
   float waveform_fps_render[SE_STATS_GRAPH_DATA];
-}se_emulator_stats_t;
+} se_emulator_stats_t;
 #define SE_FILE_BROWSER_CLOSED 0
 #define SE_FILE_BROWSER_OPEN 1
 #define SE_FILE_BROWSER_SELECTED 2
 
-typedef struct{
+typedef struct {
   char current_path[SB_FILE_PATH_SIZE];
   char file_path[SB_FILE_PATH_SIZE];
   int state; // 0 = Closed no file selected,  1= Open,  2 = closed file selected
-  tinydir_file* cached_files;
+  tinydir_file *cached_files;
   char cached_path[SB_FILE_PATH_SIZE];
   char cached_ext_filter[SB_FILE_PATH_SIZE];
   size_t num_cached_files;
-  double cached_time; 
-  tinydir_dir cached_dir; 
+  double cached_time;
+  tinydir_dir cached_dir;
   bool has_cache;
   bool allow_directory;
   unsigned num_file_types;
-  const char** file_types; 
-  void (*file_open_fn)(const char* dir);
-  char * output_path;
-}se_file_browser_state_t;
-typedef struct{
+  const char **file_types;
+  void (*file_open_fn)(const char *dir);
+  char *output_path;
+} se_file_browser_state_t;
+typedef struct {
   uint32_t turbo_toggle;
-  uint32_t hold_toggle; 
+  uint32_t hold_toggle;
   uint32_t last_turbo_toggle_presses;
   uint32_t last_hold_toggle_presses;
-}se_touch_controls_t; 
+} se_touch_controls_t;
 
-typedef struct{
+typedef struct {
   char save[SB_FILE_PATH_SIZE];
   char bios[SB_FILE_PATH_SIZE];
   char cheat_codes[SB_FILE_PATH_SIZE];
   char theme[SB_FILE_PATH_SIZE];
   char custom_font[SB_FILE_PATH_SIZE];
   char padding[3][SB_FILE_PATH_SIZE];
-}se_search_paths_t;
+} se_search_paths_t;
 
-_Static_assert(sizeof(se_search_paths_t)==SB_FILE_PATH_SIZE*8, "se_search_paths_t must contain 8 paths");
+_Static_assert(sizeof(se_search_paths_t) == SB_FILE_PATH_SIZE * 8,
+               "se_search_paths_t must contain 8 paths");
 
 #define SE_MAX_BIOS_FILES 8
 #define SE_BIOS_NAME_SIZE 32
 
 #define SE_UI_DESKTOP 0
-#define SE_UI_ANDROID 1 
-#define SE_UI_IOS     2
-#define SE_UI_WEB     3
+#define SE_UI_ANDROID 1
+#define SE_UI_IOS 2
+#define SE_UI_WEB 3
 
-typedef struct{
+typedef struct {
   char path[SE_MAX_BIOS_FILES][SB_FILE_PATH_SIZE];
   char name[SE_MAX_BIOS_FILES][SE_BIOS_NAME_SIZE];
   bool success[SE_MAX_BIOS_FILES];
-}se_bios_info_t;
+} se_bios_info_t;
 
 #define SE_MAX_CONTROL_POINTS 32
-#define SE_REGION_NAME                 0
```

Comment:
We need to make it not break aligned numbers like this.

Reply:
right I will try to see if I can make it do that